packages/@aws-cdk/aws-lambda-destinations/README.md

```diff
@@ -17,7 +17,7 @@ Destinations can be added by specifying the `onFailure` or `onSuccess` props whe
 The following destinations are supported
 
 * Lambda function
-* SQS queue
+* Standard SQS queue
```

Comment:
FIFO queues are supported for successful invocations, right?

Reply:
Yes sorry you're right 👍